install.ps1

```diff
@@ -766,70 +818,140 @@ if (-not $noGui.IsPresent) {
+        else{
+           return $false
+        }
+
+    }
+
+    function Get-ChocoPackage {
+        param (
+        [Parameter(Mandatory=$true)]
+        [string]$PackageName
+        )
+
+        choco search $PackageName -e -r | ForEach-Object {
+            $Name, $Version = $_ -split '\|'
+            New-Object -TypeName psobject -Property @{
+                'Name' = $Name
+                'Version' = $Version
+            }
+        }
+    }
+
+     function Get-VMPackage {
+        param (
+        [Parameter(Mandatory=$true)]
+        [string]$PackageName
+        )
+        if ($PackageName -notlike "*.vm") {
+            $PackageName = $PackageName + ".vm"
+        }
+        choco search $PackageName --exact -r -s "https://www.myget.org/F/vm-packages/api/v2" | ForEach-Object {
+            $Name, $Version = $_ -split '\|'
+            New-Object -TypeName psobject -Property @{
+                'Name' = $Name
+                'Version' = $Version
+            }
+        }
+    }
+
+    function Set-AdditionalPackages {
+        $additionalPackagesBox.Items.Clear()
+        foreach($package in $additionalPackages)
+        {
+            $additionalPackagesBox.Items.Add($package) | Out-Null
+        }
+    }
+
+    function Remove-SelectedPackages {
+        $additionalPackagesBox.BeginUpdate()
+        while ($additionalPackagesBox.SelectedItems.count -gt 0) {
+            $additionalPackagesBox.Items.RemoveAt($additionalPackagesBox.SelectedIndex)
+        }
+        $additionalPackagesBox.EndUpdate()
     }
 
     Add-Type -AssemblyName System.Windows.Forms
     [System.Windows.Forms.Application]::EnableVisualStyles()
-    $packagesByCategory = Get-Packages-Categories
 
     $formCategories                            = New-Object system.Windows.Forms.Form
-    $formCategories.ClientSize                 = New-Object System.Drawing.Point(1015,800)
-    $formCategories.text                       = "FLAREVM Package selection"
+    $formCategories.ClientSize                 = New-Object System.Drawing.Point(1015,850)
+    $formCategories.text                       = "FLARE-VM Package selection"
     $formCategories.StartPosition              = 'CenterScreen'
     $formCategories.TopMost                    = $true
 
     if ([string]::IsNullOrEmpty($customConfig)) {
-        $textLabel = "Select packages to install. The default configuration (recommended for reverse engineering) is pre-selected.`nClick on the reset button to restore the default configuration."
+        $textLabel = "The default configuration (recommended for reverse engineering) is pre-selected.`nClick on the reset button to restore the default configuration."
     } else {
-	    $textLabel = "Select packages to install. The provided custom configuration is pre-selected.`nClick on the reset button to restore the custom configuration."
+	    $textLabel = "The provided custom configuration is pre-selected.`nClick on the reset button to restore the custom configuration."
```

Comment:
I think we should remove the new line now that the message is shorter (as `Select packages to install` is now in a different line) and display everything in the same line.